ciel/common.py

```diff
@@ -37,8 +37,8 @@ def date_from_iso8601(string: str) -> datetime:
     return datetime.strptime(string, ISO8601_FMT)
 
 
-def mkdirp(path):
-    return pathlib.Path(path).mkdir(parents=True, exist_ok=True)
+def mkdirp(path: str) ->None: 
```

Comment:
The specification for path is that of the `pathlib.Path` constructor which you can find here: https://docs.python.org/3/library/pathlib.html#pathlib.PurePath

`str` is too restrictive.